Makefile

```diff
@@ -147,7 +147,8 @@ lb-csi-manifests: verify_image_registry deploy/k8s
 		--set kubeVersion=v1.17 \
 		--set imageRegistry=$(DOCKER_REGISTRY) \
 		--set sidecarImageRegistry=$(SIDECAR_DOCKER_REGISTRY) \
-		--set image=$(DOCKER_TAG) > deploy/k8s/lb-csi-plugin-k8s-v1.17-dc.yaml
+		--set image=$(DOCKER_TAG) \
```

Comment:
Are you moving the tagging process done by the CI into this makefile ?

Reply:
@husam-lb ?

Reply:
I don't understand the question,
but here I am adding the missing discovery client as it should have it 
deploy/k8s/lb-csi-plugin-k8s-v1.17-**dc**.yaml